### PR TITLE
feat(dataframe): implement "column.Series.from_pyarrow"

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

File: graviti/dataframe/column/series.py [hunk 1]
```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
 
         """
         return ColumnSeriesLocIndexer(self)
+
+    @classmethod
+    def _from_pyarrow(
+        cls: Type[_T], array: pa.Array, schema: PortexType, name: Optional[str] = None
+    ) -> _T:
+        obj: _T = object.__new__(cls)
+        obj._data = PagingList(array)  # type: ignore[assignment]
```
Comment (linjiX) [inline graviti/dataframe/column/series.py:241]: where the mypy error comes from ?

File: graviti/dataframe/column/series.py [hunk 2]
```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
...
+
+        Raises:
+            TypeError: When the schema mismatch with the pyarrow array type.
+
+        Returns:
+            The loaded :class:`~graviti.dataframe.column.Series` object.
+
+        """
+        if array.type != schema.to_pyarrow():
+            raise TypeError("The schema is mismatch with the pyarrow array")
```
Comment (linjiX) [inline graviti/dataframe/column/series.py:267]: `mismatched`?

Comment (linjiX) [reply graviti/dataframe/column/series.py:267]: shall we print the types to the error message?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/dataframe/column/series.py [hunk 3]
```diff
@@ -7,10 +7,16 @@
 
 
 from itertools import islice
-from typing import Any, Dict, Iterable, List, Optional, Sequence, Union, overload
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Type, TypeVar, Union, overload
```
Comment (linjiX) [inline graviti/dataframe/column/series.py:10]: in commit message `serires` -> `series`

you can just use `implement "column.Series.from_pyarrow"`

File: graviti/dataframe/column/series.py (hunk 1 above)
Comment (AChenQ) [reply graviti/dataframe/column/series.py:241]: In `__init__` function, `Series._data` has the type `Sequence[Any]`

Comment (linjiX) [reply graviti/dataframe/column/series.py:241]: shall we update the type of `_data`,
Should the `_data` always be `PagingList`?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/dataframe/column/series.py [hunk 4]
```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
...
+            name: The name to the Series.
+
+        Raises:
+            TypeError: When the schema mismatch with the pyarrow array type.
+
+        Returns:
+            The loaded :class:`~graviti.dataframe.column.Series` object.
+
+        """
+        if not array.type.equals(schema.to_pyarrow()):
```
[the remark this answers is not in the document]
Comment (AChenQ) [reply graviti/dataframe/column/series.py:266]: Yes, a little faster.

File: graviti/dataframe/column/series.py (hunk 1 above)
Comment (AChenQ) [reply graviti/dataframe/column/series.py:241]: Maybe we no need update the type now, 
We can update the type after implemented `Series.__init__` with the PagingList.

**Review by linjiX — APPROVED** [review]

File: graviti/dataframe/column/series.py [hunk 5]
```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
...
+    ) -> _T:
+        """Instantiate a Series backed by an pyarrow array.
+
+        Arguments:
+            array: The input pyarrow array.
+            schema: The schema of the series.
+            name: The name to the Series.
+
+        Raises:
+            TypeError: When the schema mismatch with the pyarrow array type.
```
Comment (Lee-000) [inline graviti/dataframe/column/series.py:260]: `When the schema is mismatched with the pyarrow array type.`?

File: graviti/dataframe/column/series.py [hunk 6]
```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
...
+
+        Returns:
+            The loaded :class:`~graviti.dataframe.column.Series` object.
```
Comment (Lee-000) [inline graviti/dataframe/column/series.py:263]: `instance` instead of `object`?

**Pull request merged**